mater/lib/src/cid.rs

```diff
@@ -3,8 +3,10 @@ use tokio::io::{AsyncRead, AsyncReadExt};
 
 use crate::{async_varint::read_varint, IDENTITY_CODE};
 
+/// Extension trait for Cid
```

Comment:
Make it clickable in the docs I guess
```suggestion
/// Extension trait for [`Cid`]
```